src/ceph-create-keys

```diff
@@ -84,12 +84,12 @@ def get_key(cluster, mon_id):
     pathdir = os.path.dirname(path)
     if not os.path.exists(pathdir):
         os.makedirs(pathdir)
-        os.chmod(pathdir, 0o770)
+        os.chmod(pathdir, 0o770 | 0770)
```

Comment:
Just trying out new things... I applying for gsoc this year under python3 support for ceph..